index.html

```diff
@@ -594,52 +596,57 @@
 
-To minimize the entanglement and side effects of these parameters, they should each affect only the corresponding aspect of the subsurface scattering, e.g. the color should not affect the size of the blur and the anisotropy should not affect the size of the blur or the color. We stipulate here a mapping which achieves these requirements and reproduces the specified RGB diffusion profile size (width on the surface) $\mathbf{r}$ and RGB multi-scatter albedo $\mathbf{A}$, by combining the empirically-derived mapping of [#Burley2018] with similarity theory. This defines in detail how the subsurface parameters collectively determine the underlying volumetric parameters (extinction $\boldsymbol{\mu}_t$, single-scattering albedo $\boldsymbol{\alpha}$, and phase function anisotropy $g$).
+The extinction coefficient is simply given by the reciprocal of the MFP per channel: $\boldsymbol{\mu}_t = 1/\mathbf{r}$. (Note, this may need to be regularized in the limit $\mathbf{r} \rightarrow 0$ to avoid numerical issues).
 
-[#Burley2018] derived (via fitting Monte Carlo simulations) an approximate relationship between the specified surface diffusion profile width $\mathbf{r}$ and multi-scatter albedo $\mathbf{A}$, and the physical extinction coefficient $\boldsymbol{\mu}_t$ which generates that profile width: $\boldsymbol{\mu}_t = 1/(S(\mathbf{A})\mathbf{r})$, where the scale-factor $S(\mathbf{A})$ is given by:
+To define the meaning of the observed **`subsurface_color`** $\mathbf{C}$ in terms of the underlying medium, the reflection from the medium needs to be separated from the Fresnel reflection at the dielectric interface. To make this precise, we denote by $\mathbf{E}_\mathrm{subsurface}$ the directional reflectance along the the normal direction (according to the definition of equation [directional-reflectance-definition]).
+This can be broken into two components:
 \begin{equation}
-S(\mathbf{A}) = 4.012 - 15.21 \mathbf{A} + 32.34 \mathbf{A}^2 - 34.68 \mathbf{A}^3 + 13.91 \mathbf{A}^4 \ .
+\mathbf{E}_\mathrm{subsurface} = \mathbf{E}_\mathrm{spec} + \mathbf{E}_\mathrm{multi-scatter} \ .
 \end{equation}
-Similarly the RGB single-scattering albedo $\boldsymbol{\alpha}$ is approximately given by
-\begin{equation}
-\boldsymbol{\alpha}(\mathbf{A}) = 1 - e^{-11.43 \mathbf{A} + 15.38 \mathbf{A}^2 - 13.91 \mathbf{A}^3} \ .
+Here $\mathbf{E}_\mathrm{spec}$ is the normal-direction reflectance of all energy reflected from the dielectric interface _without_ transmission. While $\mathbf{E}_\mathrm{multi-scatter}$ (the _multiple-scattering albedo_) is the normal-direction reflectance of all remaining reflected energy due to transmission through the interface, multiple scattering in the subsurface medium, and transmission back out.
```

Comment:
> all energy reflected from the dielectric interface _without_ transmission

Technically it's possible for light to enter and exit the medium during microfacet multiple scattering. However the distances traveled inside the medium in those cases are infinitesimal, so it seems correct for us to assume that light that reflects from the interface after this type of microfacet multiple scattering is still considered part of $\mathbf{E}_\mathrm{spec}$.

Reply:
I had in mind actually that even paths at the microsurface level that enter the medium should count as part of the multi-scatter albedo. So that the $\mathbf{E}_\mathrm{spec}$ term is rigorously composed only of paths which reflect (once or multiple times) on the microsurface but never transmit (so never touch the medium, even at the microsurface level). 

But thinking about it now, at the microsurface scale it seems appropriate to assume that the medium MFP is effectively infinite, compared to the length scale of the distances between surface scatterings (like you said). So at each macroscopic vertex in the path-trace, scattering on the microsurface generates reflection and transmission lobes, corresponding to paths which reflect or transmit to "infinity" (on the microscale) after the interaction, effectively oblivious to any volumetric media. Then the volumetric scatterings and subsequent surface bounces should happen at a larger macroscopic scale.

So I think it's more consistent then to say that the $\mathbf{E}_\mathrm{spec}$ term represents the reflection lobe of the initial macroscopic bounce  -- so it should include both single and multiple-scattering on the microsurface. The relevant distinction between $\mathbf{E}_\mathrm{spec}$ and $\mathbf{E}_\mathrm{multi-scatter}$ is just that the latter lobe consists of paths which left the initial micro-surface interaction into the lower hemisphere, so then are able to subsequently scatter in the medium (though may not, e.g. if the medium density is zero). 

This seems a tad over-academic. I'm not sure it would make much of a difference to how implementations should derive their mappings (assuming they even attempt to, rather than using one from the literature off-the-shelf such as the stated one, which is more likely). So I'll attempt to word it in a way that unambiguously describes what I wrote above, but without being overly pedantic.

Reply:
Yes, exactly, I agree with all of that reasoning.

When using approximations to microfacet multiple scattering, it's not even known what sequence of reflection and transmission events occurred at the microsurface level to produce the overall multiple-scattering reflection and transmission lobes. And when using a full random walk, the distances between microsurface events are infinitesimal anyway. So in practice it shouldn't really make any difference whether $\mathbf{E}_\mathrm{spec}$ contains secondary microfacet scattering or not.

> So I'll attempt to word it in a way that unambiguously describes what I wrote above, but without being overly pedantic.

Sounds good. A full discussion seems too academic, but since the description is pretty rigorous and technical already, it seems worthwhile to at least add a few words to clarify.

Even just adding the word "macroscopic" could potentially solve the problem (which is done like this one other place in the spec already):

"all energy reflected from the dielectric interface without (macroscopic) transmission"

Reply:
>Even just adding the word "macroscopic" could potentially solve the problem (which is done like this one other place in the spec already):

Fixed along those lines in https://github.com/AcademySoftwareFoundation/OpenPBR/pull/171/commits/47edc8cdcd80d269c946c610fe1b016bd44cf446